### ISSUE TITLE
Facet block titles display for empty facets

### ISSUE THREAD

**Issue by samhighley** [govCMS/GovCMS7#969]
This issue is impacting the ANAO website. 

It's an issue with facetapi which has been fixed here:

https://git.drupalcode.org/project/facetapi/commit/3ad60a0

Any idea when this could make it into a release? Alas it must've just missed the most recent facetapi update.

**Comment by Belhendo** [comment]
Hi @samhighley, 

As a rule, we need the changes merged into a stable release before we can use it in our distribution. 

Please raise another issue if we can help you further.
